constants/index.ts

```diff
@@ -1,131 +1,131 @@
 // NAVIGATION
 export const CATEGORY_LINKS = [
   {
-    href: "/",
-    key: "about_the_shelter",
-    label: "About the shelter",
+    href: '/',
+    key: 'about_the_shelter',
+    label: 'About the shelter',
   },
   {
-    href: "/adopt_a_paw",
-    key: "adopt_a_paw",
-    label: "Make a friend",
+    href: '/adopt_a_paw',
+    key: 'adopt_a_paw',
+    label: 'Make a friend',
   },
   {
-    href: "/create_a_paw_profile",
-    key: "create_a_paw_profile",
-    label: "Create a paw profile",
+    href: '/create_a_paw_profile',
+    key: 'create_a_paw_profile',
+    label: 'Create a paw profile',
   },
 ];
 
 export const Pets_Highlight = [
   {
     id: 1,
     image:
-      "https://www.shutterstock.com/image-photo/cute-white-kitten-blue-eyes-600nw-2253859861.jpg",
-    name: "Mimi",
-    category: "Cats",
+      'https://www.shutterstock.com/image-photo/cute-white-kitten-blue-eyes-600nw-2253859861.jpg',
+    name: 'Mimi',
+    category: 'Cats',
     age: 15,
-    gender: "Female",
-    color: "White",
-    size: "Small",
-    location: "Glebe",
-    vaccination: "Yes",
-    availability: "Yes",
+    gender: 'Female',
```

Comment:
In real life or in the future when you work with others in the team, This small changes can separate to another PR to keep PR clean. Remember: one PR one thing.

Reply:
I see. Will follow your suggestion.